packages/components/src/box-control/test/index.tsx

```diff
@@ -325,19 +286,21 @@ describe( 'BoxControl', () => {
 
 			render( <BoxControl onChange={ setState } /> );
 
+			// Typing the first letter of a unit blurs the input.
```

Comment:
Should we actually test that the focus was moved correctly?

Reply:
With ac16dd0 the test mention in the comment above is updated to run for each of the default units of `UnitControl`.